src/renderers/WebGLRenderer.js

```diff
@@ -2097,7 +2097,9 @@ class WebGLRenderer {
 
 				if ( renderTarget.isWebGLCubeRenderTarget ) {
 
-					framebuffer = __webglFramebuffer[ activeCubeFace ];
+					if ( Array.isArray( __webglFramebuffer[ activeCubeFace ] ) ) framebuffer = __webglFramebuffer[ activeCubeFace ][ activeMipmapLevel ];
```

Comment:
What app configuration requires this new code path?

Reply:
would be nice to get this  new algorithm for blended reflection captures in three